docs/config-options.md

```diff
@@ -6,6 +6,28 @@
 | `OPENVPN_USERNAME` | Your OpenVPN username             | `OPENVPN_USERNAME=asdf`                                                                                 |
 | `OPENVPN_PASSWORD` | Your OpenVPN password, beware of special charcters. Docker run vs docker-compose (using yaml) interprete special characters differently, see  [Yaml special characters](https://support.asg.com/mob/mvw/10_0/mv_ag/using_quotes_with_yaml_special_characters.htm)             | `OPENVPN_PASSWORD=asdf`                                                                                 |
 
+docker secrets are available to define OPENVPN_USER and OPENVPN_PASSWORD.
+* remove OPENVPN_USERNAME, OPENVPN_PASSWORD from environment.
+* write your credentials in two files: openvpn_user and openvpn_pass
+* add to your compose yaml:
+
+```yaml
+version: '3.8'
+services:
+ transmission:
+    ....
+    secrets:
+        - openvpn_user
+        - openvpn_pass
+
+secrets:
+    openvpn_user:
+        file: ./openvpn_user
+    openvpn_pass:
+        file: ./openvpn_pass
```

Comment:
Can’t you keep both in the same file?

Reply:
Sure, the idea was to change the least code possible. Another way is to dump secrets to credentials file if present, or env var else.

Reply:
changed in commit https://github.com/edgd1er/docker-transmission-openvpn/commit/3c78714de89419da7b7f3fd67bd368f601c73624